backend/core/views.py

```
@@ -1191,11 +1240,12 @@
             data = request.data
 
             duplicate_risk_assessment = RiskAssessment.objects.create(
-                name=data["name"],
-                description=data["description"],
-                perimeter=Perimeter.objects.get(id=data["perimeter"]),
-                version=data["version"],
+                name=data.get("name"),
+                description=data.get("description"),
+                perimeter=Perimeter.objects.get(id=data.get("perimeter")),
```

Ruff (0.11.9)

`Perimeter` may be undefined, or defined from star imports

(F405)